packages/design-tokens/design-token-previews/constants.ts

```diff
@@ -0,0 +1,70 @@
+import fontSizeTokens from "../src/tokens/font-size.tokens.json";
+import fontWeightTokens from "../src/tokens/font-weight.tokens.json";
+import colorTokens from "../src/tokens/color.tokens.json";
+import iconSizeTokens from "../src/tokens/size.tokens.json";
+import spaceTokens from "../src/tokens/space.tokens.json";
+import borderStyleTokens from "../src/tokens/border-style.tokens.json";
+import borderWidthTokens from "../src/tokens/border-width.tokens.json";
+import borderRadiiTokens from "../src/tokens/border-radius.tokens.json";
+
+import { TokenEntry } from "./types/TokenEntry";
+
+import {
+  createBorderRadiiPreview,
+  createSpacePreview,
+  createColorPreview,
+  createFontSizePreview,
+  createIconSizePreview,
+  createFontWeightPreview,
+  createBorderStylePreview,
+  createBorderWidthPreview,
+} from "./utils/previews";
+
+import { hexToHsla, hexToRgb } from "./utils/colors";
+import { getComment, getKey, getValue } from "./utils/utils";
+
+export const TOKEN_COLUMNS = {
+  [getKey(borderRadiiTokens)]: [
+    { name: "Pixels", transform: getValue },
+    { name: "Preview", transform: createBorderRadiiPreview },
```

Comment:
```typescript
const createBox = (tokenAttribute, defaultProps = {}) => ([tokenSuffix, token] => {
  const props = {
    ...defaultProps,
    [tokenAttrribute]: token.value
  };
  return <Box.div {...props}>
}
```


```suggestion
    { name: "Preview", transform: createBox('borderRaius') },
```

Reply:
Same for all the other items. You may need to create a `createText` if the `BOx.div` doesn't accept font properties. I can't remember.

Reply:
With this approach, we would only need some of the preview components. And I would consider putting them all inside a folder to nox mix with the `.mdx`